src/Session/SessionDonation/SessionObjects/Donation.php

```diff
@@ -0,0 +1,177 @@
+<?php
+namespace Give\Session\SessionDonation\SessionObjects;
+
+use DateTime;
+use Give\Helpers\ArrayDataSet;
+use Give\ValueObjects\CardInfo;
+use Give\ValueObjects\DonorInfo;
+use Give\ValueObjects\ValueObjects;
+use InvalidArgumentException;
+
+class Donation implements Objects {
```

Comment:
Please provide a docblock for this class which includes a comment explaining that this is a donation only as far as what it is being constructed in the session. It does not reflect a donation in the database. It's a pending donation.

Reply:
fixed: https://github.com/impress-org/givewp/compare/c14c8ac2fdf6...3dcd52854e4f